storm-core/src/jvm/org/apache/storm/topology/WindowedBoltExecutor.java

```diff
@@ -76,7 +76,7 @@ public WindowedBoltExecutor(IWindowedBolt bolt) {
         this.bolt = bolt;
     }
 
-    private int getTopologyTimeoutMillis(Map stormConf) {
+    private long getTopologyTimeoutMillis(Map stormConf) {
```

Comment:
TOPOLOGY_MESSAGE_TIMEOUT_SECS is integer. I don't think this method should return a long.